src/Utils/Fetcher.php

```diff
@@ -155,7 +155,13 @@ protected function getFeedUrl(){
 		$version['edition'] = $this->configReader->getEdition();
 		$version['build'] = $this->locator->getBuild();
 
-		$url = self::DEFAULT_BASE_URL . '?version=' . implode('x', $version);
+		// Read updater server URL from config
+		$updaterServerUrl = isset($this->configReader->get(['system'])['updater.server.url']) ? $this->configReader->get(['system'])['updater.server.url'] : '';
+		if($updaterServerUrl === '') {
```

Comment:
@LukasReschke yes, executing get() once according to suggestion above will be faster.